src/main/java/io/github/oliviercailloux/y2018/apartments/apartment/Apartment.java

```diff
@@ -1,136 +1,132 @@
+	 * <code>0</code> which mean the apartment doesn't have any bathroom.
 	 */
 	private int nbBathrooms;
 
 	/**
-	 * @param a boolean (true/false) which indicates if there's a terrace or not can
-	 *          be interpreted as hasTerrace. If not initialized, the parameter will
-	 *          have a default value set of <code>false</code> which mean the
-	 *          apartment doesn't have a terrace.
+	 * a boolean which indicates if there's a terrace or not. If not initialized,
+	 * the parameter will have a default value set of <code>false</code> which mean
+	 * the apartment doesn't have a terrace.
 	 */
 	private Boolean hasTerrace;
 
 	/**
-	 * @param a real number superior or equal to zero it represents the floor area
-	 *          of the terrace of the apartment if there's any. If not initialized,
-	 *          the parameter will have a default value set of <code>0</code> which
-	 *          mean the apartment have a floor area of terrace of zero, whether it
-	 *          has or not a terrace.
+	 * A real number superior or equal to zero it represents the floor area of the
+	 * terrace of the apartment if there's any. If not initialized, the parameter
+	 * will have a default value set of <code>0</code> which mean the apartment have
+	 * a floor area of terrace of zero, whether it has or not a terrace.
 	 */
 	private double floorAreaTerrace;
 
 	/**
-	 * @param a string of characters that describe the apartment and the offer (its
-	 *          accommodations). If not initialized, the parameter will have a
-	 *          default value set of <code>""</code> which mean the announcement
-	 *          doesn't have a description.
+	 * A string of characters that describe the apartment and the offer (its
+	 * accommodations). If not initialized, the parameter will have a default value
+	 * set of <code>""</code> which mean the announcement doesn't have a
+	 * description.
 	 */
 	private String description;
 
 	/**
-	 * @param a string of characters that represents the title of the announcement.
-	 *          It has to be initialized, if not, the apartment can't be created
-	 *          without a <code>title</code>.
+	 * A string of characters that represents the title of the announcement. It has
+	 * to be initialized, if not, the apartment can't be created without a
+	 * <code>title</code>.
 	 */
 	private String title;
 
 	/**
-	 * @param a boolean (true/false) which indicates if there is wireless connection
-	 *          to Internet or not can be interpreted as hasWifi. If not
-	 *          initialized, the parameter will have a default value of
-	 *          <code>false</code> which mean the apartment doesn't have wifi.
+	 * A boolean which indicates if there is wireless connection to Internet or not
+	 * can be interpreted as hasWifi. If not initialized, the parameter will have a
+	 * default value of <code>false</code> which mean the apartment doesn't have
+	 * wifi.
 	 */
 	private Boolean wifi;
```

Comment:
Ceci devrait être un `boolean`, dans ce cas.

Reply:
Bonjour, nous avions changé cela lorsque nous avons rendu les critère booléens comme obligatoire. Préférez vous un Optional ? Cela nous permettrais de vérifier si la valeur a bien été initialisée.

Reply:
Bonjour, nous avions changé cela lorsque nous avons rendu les critère booléens comme obligatoire. Préférez vous un Optional ? Cela nous permettrais de vérifier si la valeur a bien été initialisée lors de la création de l'objet.

Reply:
une solution serait de mettre des setters et des getters en `boolean` mais de laisser l'objet en Boolean au sein de la classe. Est ce que cela vous conviendrait ?

Reply:
cf https://github.com/av1m/Apartments/pull/3#discussion_r410374041

Reply:
Fixed in commit a0d76b2f15b1628af6d8570b06c8c3bee2bddb02